OasisDEX/oasis-borrow
# Ades/aave/create calculation in usdc instead of eth

# [Create calculation in USDC instead of ETH
](https://app.shortcut.com/oazo-apps/story/6424/calculations-create-calculation-in-usdc-instead-of-eth)

- Formats things in order info panel to USDC instead of ETH. 
- Provides an observable that can be reused.
  
## How to test 🧪
- Create AAVE earn position - should be no regression.
- Go to create AAVE multiply position.  It should show in USDC.
